### PR TITLE
Add PSD2 endpoint to Verify API

### PR DESCRIPTION
**Pull request by lornajane**
# Description

We have added a feature to enable secure payments confirmation for online shopping (an SCA method for the PSD2 directive). Add this to the spec.

# New 

* Added new Verify API PSD2 feature for confirming payments

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/verify.yml [hunk 1]
```diff
@@ -645,6 +645,156 @@ paths:
...
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              required:
+                - api_key
+              properties:
+                api_key:
+                  $ref: "#/components/schemas/api_key"
+                api_secret:
```
Comment (grquintero) [inline definitions/verify.yml:678]: api_secret and number would be "required" as well, no? And I don't recall if "payee" and "amount" are required as well under the PSD2 endpoint.

Comment (lornajane) [reply definitions/verify.yml:678]: Ahhh! Good catch :)

**Comment by lornajane** [comment]
I've updated to make sure the relevant fields are marked as required. If any of you are happy enough to approve this change, then I can merge and start the processes for SDKs and code snippets.

**Comment by lornajane** [comment]
Chatted with @grquintero who is happy that this is good to go

**Pull request merged**